javascript-modal/styles.css

```diff
@@ -0,0 +1,79 @@
+.container {
+  max-width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  width: 100%;
+}
+
+.modal-button {
+  position: relative;
+}
+
+.open-modal {
+  background-color: rgb(74, 73, 233);
+  color: white;
+  padding: 10px 50px;
+  font-size: 2rem;
+  border-radius: 10px;
+  border: 1px;
+  box-shadow: 0 6px 5px 0 rgba(0, 0, 0, 0.45);
+  position: absolute;
+}
+
+.overlay {
+  background-color: rgba(0, 0, 0, 0.808);
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  right: 0;
+  bottom: 0;
+}
+
+.text-box {
+  position: absolute;
+  height: 250px;
+  width: 550px;
+  background-color: white;
+  left: 28%;
+  top: 50%;
```

Comment:
Instead of needing to `absolute` position this too, you can make `.overlay` a flexbox and use `justify-content` and `align-items` to center this element 🤓